CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Next Release
 ### Internal
+- Bump Kotlin to v1.5.10
+- Bump AGP to v4.2.1
+
+## Features
+- Add NHID support in `PatientEntryScreen`
```

Comment:
Shouldn't this be [in-progress] considering you're still working on it?

Reply:
or is it a different flow?

Reply:
This is Honey's changes of adding support to show nhid. Mine is prefilling.

Reply:
https://github.com/simpledotorg/simple-android/pull/2652